ext/nnstreamer/tensor_filter/tensor_filter_qnn.cc

```diff
@@ -293,13 +293,8 @@ QnnManager::init (const char *model_path, const char *backend_path)
   }
 
   /* Create Context */
-  QnnContext_Config_t qnn_context_config = QNN_CONTEXT_CONFIG_INIT;
-  qnn_context_config.option = QNN_CONTEXT_CONFIG_OPTION_PRIORITY;
-  qnn_context_config.priority = QNN_PRIORITY_NORMAL;
-  const QnnContext_Config_t *context_configs[] = { &qnn_context_config, nullptr };
-
-  ret = qnn_interface_.contextCreate (
-      backend_handle_, nullptr, context_configs, &context_handle_);
+  const QnnContext_Config_t **context_config = nullptr;
+  ret = qnn_interface_.contextCreate (backend_handle_, nullptr, context_config, &context_handle_);
```

Comment:
doe it always null?
```
const QnnContext_Config_t *context_config = nullptr;
ret = qnn_interface_.contextCreate (backend_handle_, nullptr, &context_config, &context_handle_);

```

Reply:
It's better to set it as null. I've updated the code. PTAL